yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -289,7 +289,7 @@ function makeContractDeploymentEvent(l1BlockNum: bigint, l2Block: L2Block) {
       l2BlockNum: BigInt(l2Block.number),
       aztecAddress: extendedContractData.contractData.contractAddress.toString(),
       portalAddress: extendedContractData.contractData.portalContractAddress.toString(),
-      l2BlockHash: `0x${l2Block.getCalldataHash().toString('hex')}`,
+      l2BlockHash: `0x${l2Block.body.getCalldataHash().toString('hex')}`,
```

Comment:
Noticed that the naming became stale here. Created an issue for it: https://github.com/AztecProtocol/aztec-packages/issues/4637